app/models/katello/content_view_puppet_module.rb

```diff
@@ -17,18 +17,21 @@ class ContentViewPuppetModule < Katello::Model
     scoped_search :on => :name, :in => :content_view, :rename => :content_view_name
 
     def puppet_module
-      PuppetModule.find(self.uuid)
+      PuppetModule.find_by_uuid(self.uuid)
     end
 
     def computed_version
-      computed_version = nil
-      if Katello.config.use_elasticsearch
-        names_and_authors = [{:name => self.name, :author => self.author}]
-        search = PuppetModule.latest_modules_search(names_and_authors,
-                                           self.content_view.organization.library.repositories.puppet_type.map(&:pulp_id))
-        computed_version = search[0].version
+      if self.uuid
+        puppet_module = PuppetModule.where(:uuid => self.uuid).first
+      else
+        puppet_module = PuppetModule.latest_module(
+          self.name,
+          self.author,
+          self.content_view.puppet_repos
```

Comment:
I think this should be   'self.organization.library.repositories.puppet_type' and not the repos in the content view